app/org/sagebionetworks/bridge/dynamodb/DynamoSchedulePlanDao.java

```diff
@@ -127,13 +119,46 @@ public SchedulePlan updateSchedulePlan(StudyIdentifier studyIdentifier, Schedule
 
     @Override
     public void deleteSchedulePlan(StudyIdentifier studyIdentifier, String guid) {
+        SchedulePlan plan = getSchedulePlanInternal(studyIdentifier, guid);
+        if (plan != null) {
+            plan.setDeleted(true);
+            mapper.save(plan);
+        }
+    }
```

Comment:
Unit test that deleting a non-existent plan silently does nothing? (For both logical and physical cases.)